components/ui/GenericAvatar.tsx

```diff
@@ -43,6 +44,7 @@ export default function GenericAvatar({ className, name, size }: Props) {
       className={classNames(className, 'rounded')}
       style={{
         backgroundColor: color,
+        borderRadius: round ? '50%' : undefined,
```

Comment:
does the `undefined` value override the `.rounded` class radius ?

Reply:
It would also be good to add a test for that new prop

Reply:
No problem regarding the `.rounded` class border-radius as the element border-radius is not even there when using undefined. I just tested it and there's no problem there.

Reply:
Nice :)

Reply:
I just added the missing test 👍